src/test/java/wooteco/subway/line/LineAcceptanceTest.java

```diff
@@ -0,0 +1,163 @@
+        ExtractableResponse<Response> createResponse1 = createLine(lineRequest1);
+
+        Map<String, String> params2 = new HashMap<>();
+        params2.put("name", "역삼역");
+        ExtractableResponse<Response> createResponse2 = RestAssured.given().log().all()
+                .body(params2)
+                .contentType(MediaType.APPLICATION_JSON_VALUE)
+                .when()
+                .post("/stations")
+                .then().log().all()
+                .extract();
+
+        // when
+        ExtractableResponse<Response> response = RestAssured.given().log().all()
+                .when()
+                .get("/lines")
+                .then().log().all()
+                .extract();
+
+        // then
+        assertThat(response.statusCode()).isEqualTo(HttpStatus.OK.value());
+        List<Long> expectedLineIds = Stream.of(createResponse1, createResponse2)
+                .map(it -> Long.parseLong(it.header("Location").split("/")[2]))
+                .collect(Collectors.toList());
+        List<Long> resultLineIds = response.jsonPath().getList(".", LineResponse.class).stream()
+                .map(LineResponse::getId)
+                .collect(Collectors.toList());
+        assertThat(resultLineIds).containsAll(expectedLineIds);
+    }
+
+    @DisplayName("지하철 노선 1개를 보여준다.")
+    @Test
+    void showLine() {
+        // given
+
+        LineRequest lineRequest1 = new LineRequest("2호선", "bg-red-600", 0L, 0L, 0);
+        ExtractableResponse<Response> createResponse = createLine(lineRequest1);
+
+        // when
+        String uri = createResponse.header("Location");
+        ExtractableResponse<Response> response = RestAssured.given().log().all()
+                .when()
+                .get(uri)
+                .then().log().all()
+                .extract();
+
+        // then
+        assertThat(response.statusCode()).isEqualTo(HttpStatus.OK.value());
+        Long expectedLineId = Long.parseLong(createResponse.header("Location").split("/")[2]);
+        Long resultLineId = response.jsonPath().getObject(".", LineResponse.class).getId();
+        assertThat(resultLineId).isEqualTo(expectedLineId);
+    }
+
+    @DisplayName("지하철 노선을 삭제한다.")
+    @Test
+    void deleteLine() {
+
+        // given
+        LineRequest lineRequest1 = new LineRequest("2호선", "bg-red-600", 0L, 0L, 0);
+        createLine(lineRequest1);
+
+        LineRequest lineRequest2 = new LineRequest("3호선", "bg-red-600", 0L, 0L, 0);
+        ExtractableResponse<Response> createResponse = createLine(lineRequest2);
+
+        // when
+        String uri = createResponse.header("Location");
+        ExtractableResponse<Response> response = RestAssured.given().log().all()
+                .when()
+                .delete(uri)
+                .then().log().all()
+                .extract();
+
+        // then
+        assertThat(response.statusCode()).isEqualTo(HttpStatus.NO_CONTENT.value());
```

Comment:
```suggestion
                .then().log().all()
                .statusCode(HttpStatus.NO_CONTENT.value());
```

응답 코드만 확인할 거라면 굳이 응답을 추출하지 않고 이렇게 해도 될 것 같아요. 다만, 테스트 전체적으로 일관성을 가져갈 수 있게 한 가지 방법을 택하는 것이 좋을 것 같네요. 지금은 전체적으로 테스트 코드가 정돈되지 않은 느낌이 들어요.

Reply:
사실 RestAssured를 사용해본적이 없어서 아직 익숙하지 않았던 것 같아요!
그래서 그때 그때 찾고 맞추어서 하다보니 정돈되지 않았던 것 같네요..!